pkg/child/child.go

```diff
@@ -484,3 +518,16 @@ func (e *reaperErr) Error() string {
 	}
 	return fmt.Sprintf("exited with WAITSTATUS=0x%08x", e.ws)
 }
+
+func NewNetNsWithPathWithoutEnter(p string) error {
+	if err := os.WriteFile(p, nil, 0400); err != nil {
+		return err
+	}
+	// this is hard (not impossible though) to reimplement in Go: https://github.com/cloudflare/slirpnetstack/commit/d7766a8a77f0093d3cb7a94bd0ccbe3f67d411ba
+	cmd := exec.Command("unshare", "-n", "mount", "--bind", "/proc/self/ns/net", p)
```

Comment:
agree. It was hard to implement this in my previous PR too but you the link example should help us ;)